### PR TITLE
INTG-2820 fix reports

### PR DESCRIPTION
**Pull request by MickStanciu**
## Summary

apply folder check to reports and sqlite

## Checklist

- [x] Is this PR a reasonable size?

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

File: frontend/src/routes/Config.svelte [hunk 1]
```diff
@@ -367,14 +366,36 @@
 					})
 					break
 				case 'reports':
-					$exportConfig['items'] = ['inspections','reports']
-					ExportReports()
-					push("/export/status")
+					ValidateExportDirectory().then((result) => {
+						if (result === true) {
+							showBanner = false
```
Comment (sc-gv) [inline frontend/src/routes/Config.svelte:371]: can we wrap this code in a function and maybe pass a flag to do the associated export?

**Review by sc-gv — APPROVED** [review]

File: frontend/src/routes/Config.svelte (hunk 1 above)
Comment (MickStanciu) [reply frontend/src/routes/Config.svelte:371]: we can. But wanted to limit the risk for now. We can definitely optimize this as a fix-it-friday thing
